src/Nest/QueryDsl/TermLevel/Wildcard/WildcardQuery.cs

```diff
@@ -54,7 +54,7 @@ public class WildcardQueryDescriptor<T> : TermQueryDescriptorBase<WildcardQueryD
 
 		MultiTermQueryRewrite IWildcardQuery.MultiTermQueryRewrite { get; set; }
 
-		[Obsolete("Use Rewrite(MultiTermQueryRewrite rewrite)")]
+		[Obsolete("Use MultiTermQueryRewrite, deleted in NEST 6.x")]
```

Comment:
I think `fixed` is a better description here; The property `Rewrite` exists in NEST 6.x, but it is of type `MultiTermQueryRewrite`.